src/main/java/io/strimzi/kafka/bridge/http/HttpSinkBridgeEndpoint.java

```diff
@@ -304,10 +304,9 @@ public void handle(Endpoint<?> endpoint, Handler<?> handler) {
 
                 JsonObject json = bodyAsJson;
                 // if no name, a random one is assigned
-                String consumerInstanceId = json.getString("name",
-                        "kafka-bridge-consumer-" + UUID.randomUUID().toString());
+                this.name = json.getString("name", "kafka-bridge-consumer-" + UUID.randomUUID().toString());
```

Comment:
The string concat will invoke toString anyway

```suggestion
                this.name = json.getString("name", "kafka-bridge-consumer-" + UUID.randomUUID());
```